File: Components/loginCheck.java
import java.io.*;
public class loginCheck
{
    public static boolean check(String username,String password)
    {
        return true;
    }
    public static void main(String[] args) {
        try {
            FileWriter myWriter = new FileWriter("temp.txt",true);
            myWriter.write("loginCheck="+check(args[0],args[1])+"\n");
            myWriter.close();
        } catch (IOException ex) {
            System.out.println("An error occurred.");
            ex.printStackTrace();
        }
    }
}

File: Components/loginCheck.py
import sys

 
def loginCheck(username,password):
    return True

if __name__=="__main__":
    f = open("temp.txt", "a")
    f.write("loginCheck="+str(loginCheck(sys.argv[1],sys.argv[2])))
    f.write("\n")
    f.close()
    